Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemController.java

```diff
@@ -101,6 +104,12 @@ public int saveProblemWithTitleAndGetId(@RequestBody String title) {
         return saveProblemWithTitleUseCase.execute(title);
     }
 
+    @PatchMapping(value = "/{problemId}",
+            consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
`consumes = MediaType.APPLICATION_JSON_VALUE`  this is unnecessary

default type is JSON already